openff/recharge/cli/reconstruct.py

```diff
@@ -59,7 +57,7 @@ def _process_result(
     result_tuple: tuple[
         "qcportal.record_models.BaseRecord",
         "qcelemental.models.Molecule",
-        "qcportal.models.KeywordSet",
```

Comment:
could we take the record here and then extract the molecule and keywords using `record.molecule` and `record.specification.keywords`?